packages/web-components/src/common/utils/__mocks__/mock-google-map.ts

```diff
@@ -0,0 +1,386 @@
+const createMockFuncsFromArray = (instance: any, names: string[] = []) => {
+  names.forEach(name => {
+    instance[name] = jest.fn().mockName(name)
+  })
+}
+
+const createGoogleMapsMock = (libraries = []) => {
+  const createMVCObject = (instance: any) => {
+    const listeners: any = {}
+    instance.listeners = listeners
+
+    instance.addListener = jest
+      .fn((event, fn) => {
+        listeners[event] = listeners[event] || []
+        listeners[event].push(fn)
+        return {
+          remove: () => {
+            const index = listeners[event].indexOf(fn)
+
+            if (index !== -1) {
+              listeners[event].splice(index, 1)
+            }
+          },
+        }
+      })
+      .mockName('addListener')
+
+    createMockFuncsFromArray(instance, ['bindTo', 'get', 'notify', 'set', 'setValues', 'unbind', 'unbindAll'])
+  }
+
+  const maps = {
```

Comment:
This mock gets loaded in the jest-setup file at the root of the project - might be not necessary at all!